src/lib.rs

```diff
@@ -214,3 +216,36 @@ pub fn parse(text: &str, options: LiquidOptions) -> Result<Template> {
     let tokens = try!(lexer::tokenize(&text));
     parser::parse(&tokens, &options).map(Template::new)
 }
+
+/// Parse a liquid template from a file, returning Result<Template, io::Error>
+/// # Examples
+///
+/// ## Minimal Template
+///
+/// template.liquid:
+/// ```
+/// "Liquid! {{num | minus: 2}}"
+/// ```
+///
+/// Your rust code:
+/// ```
+/// use liquid::{Renderable, LiquidOptions, Context};
+///
+/// let template = liquid::parse_file("/path/to/template.liquid",
+///                                    LiquidOptions::default()).unwrap();
+/// let mut data = Context::new();
+/// data.set_val("num", Value::Num(4f32));
+/// let output = template.render(&mut data);
+/// assert_eq!(output.unwrap(), Some("Liquid! 2".to_string()));
+/// ```
+pub fn parse_file(fp: &str, options: LiquidOptions) -> Result<Template> {
```

Comment:
You can use [AsRef](https://doc.rust-lang.org/std/convert/trait.AsRef.html) to enable automatic conversion of supported types to `Path`, which allows for a more flexible API. Basically just copy the function signature of [File::open](https://doc.rust-lang.org/std/fs/struct.File.html#method.open).